docs/architecture/adr-017-chain-versions.md

```diff
@@ -91,10 +90,4 @@ When a chain forks or upgrades but continues the same history, it takes a new Ch
 ChainDescription = <ChainID>/x/<Height>/<ForkDescription>
 ```
 
-Where
-    - ChainID is the ChainID from the previous ChainDescription (ie. its hash)
-    - `x` denotes that a change occured
-    - `Height` is the height the change occured
-    - ForkDescription has the same form as ChainDescription but for the fork
-        - this allows forks to specify new versions for tendermint or the app, as well as arbitrary changes to the state or validator set
-
+Where - ChainID is the ChainID from the previous ChainDescription (ie. its hash) - `x` denotes that a change occured - `Height` is the height the change occured - ForkDescription has the same form as ChainDescription but for the fork - this allows forks to specify new versions for tendermint or the app, as well as arbitrary changes to the state or validator set
```

Comment:
I'm confused, we want this to be a list

Reply:
Yeah, the Prettier linter formatted the list into a paragraph because the word `Where` is breaking the parser. We'd have to fix this one manually.